### PR TITLE
Added company count to the affiliate backend

### PR DESCRIPTION
**Pull request by bitboxer**
Closes https://github.com/railslove/recover-backlog/issues/152

### CODE REVIEW

File: app/models/concerns/rails_admin_config/for_affiliate.rb [hunk 1]
```diff
@@ -16,6 +16,12 @@ module ForAffiliate
             bindings[:view].link_to(bindings[:object].owner_count, "#{path}?f[affiliate][1][o]=is&f[affiliate][1][v]=#{ERB::Util.url_encode(bindings[:object].code)}")
           end
         end
+        field :companies_count do
+          label "Companies"
+          formatted_value do
+            bindings[:object].company_count
+          end
```
Comment (salimhb) [inline app/models/concerns/rails_admin_config/for_affiliate.rb:23]:
```suggestion
          end
          read_only true
```

File: app/models/concerns/rails_admin_config/for_affiliate.rb [hunk 2]
```diff
@@ -16,6 +16,12 @@ module ForAffiliate
             bindings[:view].link_to(bindings[:object].owner_count, "#{path}?f[affiliate][1][o]=is&f[affiliate][1][v]=#{ERB::Util.url_encode(bindings[:object].code)}")
           end
```
Comment (salimhb) [inline app/models/concerns/rails_admin_config/for_affiliate.rb:17]:
```suggestion
          end
          read_only true
```

**Review by salimhb — CHANGES_REQUESTED** [review]
virtual fields should me marked as readonly, otherwise they will show up as editable in RailsAdmin. This prevents us from saving an Affiliate as it tries to update these non-existing fields.
image

**Comment by bitboxer** [comment]
> virtual fields should me marked as readonly, otherwise they will show up as editable in RailsAdmin. This prevents us from saving an Affiliate as it tries to update these non-existing fields.

Oh, sorry. Totally forgot that part. Thanks for finding it.

**Review by salimhb — APPROVED** [review]

**Pull request merged**